tests/integration/conftest.py

```diff
@@ -10,7 +10,7 @@
 from pathlib import Path
 from typing import List
 
-import kubernetes
+import kubernetes  # type: ignore[import]
```

Comment:
Similar to earlier, this could potentially be resolved if `kubernetes` is added to the `deps` of the `[testenv:lint]` environment in `tox.ini`

Reply:
Without this, I get the error "error: Skipping analyzing "kubernetes": module is installed, but missing library stubs or py.typed marker  [import]". 
It seems that kubernetes doesn't have type stubs, if I understood correctly. 
We might consider using the plugin https://pypi.org/project/kubernetes-typed/ in a further PR.

Reply:
ISD-422 was created to address type hints.